Microsoft.Azure.Devices.Edge.sln

```diff
@@ -197,11 +197,15 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Edg
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "edge-modules", "edge-modules", "{69157D2B-2AD3-4EE0-A837-6AD9329EDDC3}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TwinTester", "edge-modules\TwinTester\TwinTester.csproj", "{9237E577-DD0B-4A49-8155-7BC8635F9DBD}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TwinTester", "edge-modules\TwinTester\TwinTester.csproj", "{9237E577-DD0B-4A49-8155-7BC8635F9DBD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ModuleRestarter", "edge-modules\ModuleRestarter\ModuleRestarter.csproj", "{2C7AE89A-648C-4EBC-8084-0E59BAFF496D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ModuleRestarter", "edge-modules\ModuleRestarter\ModuleRestarter.csproj", "{2C7AE89A-648C-4EBC-8084-0E59BAFF496D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Relayer", "edge-modules\Relayer\Relayer.csproj", "{C1FDA6CF-7EF6-4C3F-81E7-4C13EE399BF1}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Relayer", "edge-modules\Relayer\Relayer.csproj", "{C1FDA6CF-7EF6-4C3F-81E7-4C13EE399BF1}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "modules", "modules", "{F921339B-32F9-4BF3-B364-2DB01FA2F1A1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestResultCoordinator", "test\modules\TestResultCoordinator\TestResultCoordinator.csproj", "{8181EB49-62CE-495B-8078-08DCF8C30541}"
```

Comment:
Everything in this file is auto-generated. I didn't touch any of the other modules or the file structure